Makefile.in

```diff
@@ -360,8 +370,8 @@ maintainerclean: distclean
 # implicit rules
 #
 .c.$(OBJEXT):
-	dir="$$(dirname $@)"; [ -d "$$dir" ] || $(MKDIR_P) "$$dir"
-	$(CC) -I. -I$(srcdir) -I$(srcdir)/main $(DEFS) $(ALL_CPPFLAGS) $(ALL_CFLAGS) -o $@ -c $<
+	@dir="$$(dirname $@)"; [ -d "$$dir" ] || $(MKDIR_P) "$$dir"
+	$(V_CC) -I. -I$(srcdir) -I$(srcdir)/main $(DEFS) $(ALL_CPPFLAGS) $(ALL_CFLAGS) -o $@ -c $<
```

Comment:
if you want the mkidr part to follow visibility of the `CC` line, you could reword this if `V_CC` didn't include `CC`, as mentioned above.
Something like this (untested):
```make
.c.$(OBJEXT):
	$(V_CC); \
	dir="$$(dirname $@)"; ([ -d "$$dir" ] || $(MKDIR_P) "$$dir") && \
	$(CC) -I. -I$(srcdir) -I$(srcdir)/main $(DEFS) $(ALL_CPPFLAGS) $(ALL_CFLAGS) -o $@ -c $<
```